source/ControllableHero.hx

```diff
@@ -0,0 +1,48 @@
+package;
+
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.FlxG;
+
+class ControllableHero extends FlxSprite
+{
+    public function new()
+        {
+            super();
+            makeGraphic(50, 45, FlxColor.WHITE, false);
+        }
+
+
+        override public function update(elapsed:Float) 
+        {
+            if (FlxG.keys.justPressed.UP)
+                {this.y += -1;}
+            if (FlxG.keys.pressed.UP)
+                {this.y += -1;}
+            if (FlxG.keys.justReleased.UP)
+                {this.y += -1;}
+        
+            if (FlxG.keys.justPressed.RIGHT)
+                {this.x += 1;}
+            if (FlxG.keys.pressed.RIGHT)
+                {this.x += 1;}
```

Comment:
Also, to note: this is nice and functional, but it does fall prey to the problem we discussed in class the other day - update calls happen every time the screen is drawn - if we run at a very, very fast framerate our character is going to move 1 pixels for every one of those visual frames.

No change needed here (unless you want to try to solve this problem) we'll discuss it more in class how we can work around the issue.